keyboards/tweetydabird/lotus58/atmeldfu/lotus58.c

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 Markus Knutsson (@TweetyDaBird)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
```

Comment:
This should be moved to the top level keyboard folder (`keyboards/tweetydabird/lotus58`), and it removed from the rest of the sub folders.